ashlar/imports/shapefile.py

```diff
@@ -25,17 +25,6 @@ def get_shapefiles_in_dir(path):
     return shapefiles
 
 
-def get_union(geoms):
-    """Attempts to union a list of geometries."""
-    if len(geoms) <= 0:
-        raise ValueError('Cannot union empty feature list.')
-
-    combined = geoms[0]
-    for geom in geoms[1:]:
-        combined = combined.union(geom)
-    return combined
```

Comment:
The `get_union` method was untested, and was also not called anywhere in the codebase.